src/proto/mgmt/storage.proto

```diff
@@ -121,6 +118,67 @@ message ScmModule {
 	//string fwrev = 10;
 }
 
+enum ResponseStatus {
+	SUCCESS = 0;
+	IN_PROGRESS = 1;
+	WAITING = 2;
+	ERR_APP = -1;			// Application error
+	ERR_UNKNOWN = -2;		// Unknown error
+}
+
+message ResponseState {
+	ResponseStatus status = 1;
+	string error = 2;
+	string info = 3;
+}
+
+message NvmeControllerResp {
+	string pciaddr = 1;		// PCI address of NVMe controller
+	ResponseState state = 2;
+}
+
+message ScmMountResp {
+	string mntpoint = 1;		// Whene device or tmpfs is mounted
+	ResponseState state = 2;
+}
+
+message ScmModuleResp {
+	string physicalid = 1;		// SCM module identifier
```

Comment:
Is this the same as the physical ID coming from ipmctl? In that case I think it is the SMBIOS type 17 table handle, which supplies no real information about the device location. Might be more useful to identify either by socket/controller/slot or serial number.

Reply:
I think I will just use serial number unless we want to actually embed the module message itself here

Reply:
wouldn't you need socket/controller/channel/slot?

Reply:
In my experience the motherboard labeling doesn't match up, so knowing socket/controller/channel/slot is only helpful if you've internalized the mapping. Including both might be helpful - so if the user knows the mapping, they can find the module quickly, but at the very least the user can confirm the serial.

Reply:
that's useful advice, will include both